calamari.suse.spec

```diff
@@ -0,0 +1,111 @@
+#
+# spec file for package calamari
+#
+# Copyright (c) 2014 SUSE LINUX Products GmbH, Nuernberg, Germany.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via http://bugs.opensuse.org/
+#
+
+
+Name:           calamari
+Summary:        Manage and monitor Ceph with a REST API
+License:        LGPL-2.1+
+Group:          System/Filesystems
+Version:        1.3+git.1456160171.f7475e3
+Release:        0
+Url:            http://ceph.com/
+Source0:        %{name}-%{version}.tar.gz
+Provides:       calamari-server = %{version}
+Obsoletes:      calamari-server < %{version}
+# Force no database usage
+Conflicts:      python-SQLAlchemy
+Conflicts:      python-alembic
+Requires:       logrotate
+Requires:       python-django < 1.7
+Requires:       python-djangorestframework
+Requires:       python-dateutil
+Requires:       python-zerorpc
+Requires:       python-gevent >= 1.0
+Requires:       python-pytz
+Requires:       python-setuptools
+%{?systemd_requires}
+BuildRequires:  fdupes
+BuildRequires:  python-devel
+BuildRequires:  systemd
+# For lsb_release binary
+BuildRequires:  lsb-release
+BuildRequires:  python-setuptools
+# For /etc/*release files
+%if 0%{?suse_version} == 1315 && (! 0%{?is_opensuse})
+BuildRequires:  sles-release
+%else
+BuildRequires:  suse-release
+%endif
+BuildArch:      noarch
+
+%description
+Calamari is a REST API for monitoring and controlling a Ceph cluster.
+It is intended to be used by other frontent GUIs.
+
+%prep
+%setup -q
+
+%build
+echo "VERSION =\"%{version}\"" > rest-api/calamari_rest/version.py
+
+%install
+make DESTDIR=%{buildroot} install-lsb
+mkdir -p %{buildroot}%{_sbindir}
+ln -s -f %{_sbindir}/service %{buildroot}%{_sbindir}/rccalamari
+%fdupes %{buildroot}%{python_sitelib}
+
+%files
+%defattr(-,root,root,-)
+%doc COPYING*
+%exclude /opt/calamari
+%exclude %{_bindir}/calamari-ctl
+%exclude %{_bindir}/cthulhu-manager
+%exclude %{_sysconfdir}/calamari/alembic.ini
+%exclude %{_sysconfdir}/graphite
+%exclude %{_sysconfdir}/salt
+%exclude %{_sysconfdir}/supervisor
+%{python_sitelib}/calamari_*
+%{python_sitelib}/cthulhu*
+%{_bindir}/calamari-lite
+%{_unitdir}/calamari.service
+%{_sbindir}/rccalamari
```

Comment:
Yeah, /sbin/rcFOO files are symlinks to either sysvinit scripts, or to /sbin/service for systemd unit files. It's a convenience thing, so that instead of having to go to all the trouble(!) of typing `service FOO start` or `systemctl start FOO`, you can instead type `rcFOO start`.